topology/University of Florida/UF CMS Tier2/UFlorida-PG_downtime.yaml

```diff
@@ -1,42 +1 @@
-- Class: UNSCHEDULED
-  Description: |-
-    One of the Lustre OSSes went down.
-    The OSS can be recovered in the morning, hopefully.
-  EndTime: Nov 30, 2016 14:00 +0000
-  ID: 1005134
-  ResourceName: UFlorida-SE
-  Services:
-  - SRMv2
-  Severity: Outage
-  StartTime: Nov 30, 2016 05:00 +0000
-# ----------------------------------------------------------
-- Class: UNSCHEDULED
-  Description: SE load became high due to a high read-write activity.
-  EndTime: Dec 4, 2016 14:00 +0000
-  ID: 1005136
-  ResourceName: UFlorida-SE
-  Services:
-  - SRMv2
-  Severity: Outage
-  StartTime: Dec 3, 2016 05:00 +0000
-# ----------------------------------------------------------
-- Class: UNSCHEDULED
-  Description: One of lustre cmsoss is down due to a dead RAID controller.
-  EndTime: Feb 7, 2017 22:00 +0000
-  ID: 1005198
-  ResourceName: UFlorida-SE
-  Services:
-  - SRMv2
-  Severity: Outage
-  StartTime: Feb 7, 2017 17:00 +0000
-# ----------------------------------------------------------
-- Class: UNSCHEDULED
-  Description: One of OSSes became unavailable.
-  EndTime: May 29, 2017 14:00 +0000
-  ID: 1005324
-  ResourceName: UFlorida-SE
-  Services:
-  - SRMv2
-  Severity: Outage
-  StartTime: May 29, 2017 13:00 +0000
-# ----------------------------------------------------------
+
```

Comment:
Looks like you replaced `topology/University of Florida/UF CMS Tier2/UFlorida-PG_downtime.yaml` with a single blank line -- but you need to remove the file.

Eg,
`git rm -f "topology/University of Florida/UF CMS Tier2/UFlorida-PG_downtime.yaml"`

Reply:
That's my bad, I thought empty downtime files were ok

Reply:
so all the empty _downtime file should be removed?

Reply:
Yep!

Reply:
So for the merge conflict with UChicago file, how should I do? I find it is a "Disable: true" group and just deleted this file.

Reply:
It looks like the conflict for the UChicago file is from your f1195ae0ddaf891b671509e037066f9e24acc06a, which just removes the OSG VOOwnership.

Anyway, to resolve the conflict you can just remove (git rm -f) the file.

Reply:
OK, I have removed that file in my previous branch